### PR TITLE
Add and configure the accessibility testing tool pa11y-ci

### PR DESCRIPTION
**Pull request by staceybeard**
# Description

I've added the development dependency pa11y-ci to be used to run accessibility testing on Quill.
- Added pa11y-ci and a script to run it in `package.json`.
- Configured pa11y-ci in `.pa11yci` by specifying which URLs to check and using Actions to log in.
- Added documentation on pa11y-ci to CONTRIBUTING.md.
- Added the pa11y-ci script to run during the "Lint, Build and Run" GitHub Action **[status: removed]**.

## Motivation and Context

Adding an accessibility checker to this project should make contributors more aware of issues relating to accessibility. Addressing the errors raised by pa11y-ci will improve Quill's accessibility, ensuring that all hackathon enthusiasts can use Quill, regardless of ability.

## How Has This Been Tested

- The npm script for pa11y-ci was run several times to make sure it's working correctly.

## Screenshots (if appropriate)
npm script (the errors shown mean that it's working)

pa11y-success

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by krubenok — CHANGES_REQUESTED** [review]
We need to setup the test: accessibility script to also spawn the application itself *then* run the tests. Ideally it should not be reliant on anything outside of that command.

**Comment by krubenok** [comment]
This should be edited to reflect that it's local only. @staceybeard

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,6 +50,34 @@ Any changes to the code base should follow the style and coding conventions
...
+
+If your contribution adds any new pages to Quill, please add them to 
+`.pa11yci` to make sure that these pages are covered by the accessibility 
+tests. If your new pages are accessed as a non-logged-in user, add them at the 
+beginning of the URL list. If they are accessed when logged in, add them after 
+the URL with actions to log in.
+
+For more information on pa11y-ci, please visit [pa11y-ci] and [pa11y], 
+in particular the [section on actions][pa11y-actions].
+
```
Comment (krubenok) [inline CONTRIBUTING.md]: I think these comments should probably be in the README.md file, not in contributing. @staceybeard

**Review by krubenok — CHANGES_REQUESTED** [review]
Please update documentation!

File: CONTRIBUTING.md (hunk 1 above)
Comment (staceybeard) [reply CONTRIBUTING.md]: Sure, I'll move them.

Comment (staceybeard) [reply CONTRIBUTING.md]: Done

**Review by krubenok — APPROVED** [review]

**Pull request merged**